Directory.Build.props

```diff
@@ -8,6 +9,9 @@
     <!-- Suppress SYSLIB0020 for generated System.Text.Json code that uses obsolete IgnoreNullValues -->
     <NoWarn>$(NoWarn);SYSLIB0020</NoWarn>
     <BuildNative Condition="'$(BuildNative)' == ''">false</BuildNative>
+    <!-- Identify Server Projects -->
+    <_RelativeProjectDir>$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)').Replace($([System.IO.Path]::GetFullPath('$(RepoRoot)')), ''))</_RelativeProjectDir>
```

Comment:
It may be easier to test for the `$(CliName)` property

Reply:
i.e.   There are a handful of properties that will exist in all server projects but not in any tool or test project